plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/schema/confluent/TestConfluentSchemaRegistryTableDescriptionSupplier.java

```diff
@@ -92,6 +95,29 @@ public void testCaseInsensitiveSubjectMapping()
                         Optional.empty()));
     }
 
+    @Test
+    public void testAmbiguousSubject()
+            throws Exception
+    {
+        TableDescriptionSupplier tableDescriptionSupplier = createTableDescriptionSupplier();
+        String topicName = "topic_one";
+        SchemaTableName schemaTableName = new SchemaTableName(DEFAULT_NAME, topicName);
+
+        SCHEMA_REGISTRY_CLIENT.register(topicName + "-key", getAvroSchema(schemaTableName.getTableName(), ""));
+        SCHEMA_REGISTRY_CLIENT.register(topicName.toUpperCase(ENGLISH) + "-key", getAvroSchema(schemaTableName.getTableName(), ""));
+
+        assertTrue(tableDescriptionSupplier.listTables().contains(schemaTableName));
+
+        assertThatThrownBy(() ->
+                tableDescriptionSupplier.getTopicDescription(
+                        TestingConnectorSession.builder()
+                                .setPropertyMetadata(new ConfluentSessionProperties(new ConfluentSchemaRegistryConfig()).getSessionProperties())
+                                .build(),
+                        schemaTableName))
+                .isInstanceOf(TrinoException.class)
+                .hasMessage("Ambiguous subjects TOPIC_ONE,topic_one");
```

Comment:
`Unable to access 'topic_one' table. Subject is ambiguous, and may refer to one of the following: TOPIC_ONE, topic_one`?